packages/firebase/index.d.ts

```diff
@@ -7617,6 +7617,19 @@ declare namespace firebase.storage {
     md5Hash?: string | null;
   }
 
+  /**
+   * An error returned by the Firebase Storage SDK.
+   */
+  interface FirebaseStorageError extends FirebaseError {
+    customData: { serverResponse: string | null };
```

Comment:
Since you have a getter `serverResponse`, `customData` feels like the internal implementation and you can just expose the getter.

Reply:
Done.